lxd/instance/instancetype/instance.go

```diff
@@ -1210,6 +1210,14 @@ func ConfigKeyChecker(key string, instanceType Type) (func(value string) error,
 			return validate.IsAny, nil
 		}
 
+		if strings.HasSuffix(key, ".bus") {
```

Comment:
Please can you add the lxdmeta:generate documentation for these 2 new keys. Thanks